e2e/index.js.md

```diff
@@ -16,8 +40,8 @@ Generated by [AVA](https://ava.li).
       docsify start <path>  Server for SSR␊
     ␊
     Global Options␊
-      --help, -h     Show help                                             [boolean]␊
-      --version, -v  Show version number                                   [boolean]␊
+      --help, -h     Show help                                             [booléen]␊
```

Comment:
Can you please check why its coming 
`[booléen]␊`

the previous one was fine.

Reply:
Actually I'm investigating it because I have no clue why french is printed instead of english

Reply:
I guess no need to commit the snapshots

cc @jamesgeorge007 can you take a look why its happening

Reply:
Perhaps the types are translated according to the locale of the client - I'm in France which might explain why

Reply:
I'll remove this commit

Reply:
ok then it should not update the snapshots, it should throw error.
Something is wrong I guess with the test setup or with the npm script

Reply:
Yeah it isn't intended.

Reply:
I guess no need to commit the snapshots....